common/src/main/java/com/wynntils/features/chat/MessageFilterFeature.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.chat;
+
+import com.wynntils.core.config.Category;
+import com.wynntils.core.config.Config;
+import com.wynntils.core.config.ConfigCategory;
+import com.wynntils.core.config.RegisterConfig;
+import com.wynntils.core.features.Feature;
+import com.wynntils.core.text.CodedString;
+import com.wynntils.handlers.chat.event.ChatMessageReceivedEvent;
+import com.wynntils.handlers.chat.type.MessageType;
+import com.wynntils.utils.type.Pair;
+import java.util.List;
+import java.util.regex.Pattern;
+import net.minecraft.ChatFormatting;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+@ConfigCategory(Category.CHAT)
+public class MessageFilterFeature extends Feature {
+    // List of Pair<Foreground, Background>
+    // Ensures we only try relevant regexes for any given message
+
+    private static final List<Pair<Pattern, Pattern>> WELCOME = List.of(
+            Pair.of(Pattern.compile("^§7Loading Resource Pack\\.\\.\\.$"), null),
+            Pair.of(Pattern.compile("^§6Thank you for using the WynnPack\\. Enjoy the game!$"), null),
+            Pair.of(
+                    Pattern.compile(
+                            "^§cSelect a character! Each character is saved individually across all servers, you can come back at any time with /class and select another character!$"),
+                    null),
```

Comment:
This can be useful and you disable it for everyone by default. Maybe we should make this feature disabled by default?

Reply:
I will disable by default but I feel like it is a welcome message, since it always comes with the multiline welcome after you select a class. You can't actually see this message with the custom character selector until *after* you already selected a character,